### DISCUSSION TITLE
Access to wire friend list through e2

### DISCUSSION THREAD

**Discussion by Isemenuk27** [wiremod/wire#2814] [Suggestions]
will be useful for single chip contraptions

**Comment by Denneisk** [comment]
This is relatively trivial, [this](https://github.com/wiremod/wire/blob/master/lua/wire/stools/friendslist.lua#L52) just needs to be exposed.

Looking ahead, friendslists are very wasteful.
https://github.com/wiremod/wire/blob/master/lua/wire/stools/friendslist.lua#L56-L66
https://github.com/wiremod/wire/blob/master/lua/entities/gmod_wire_friendslist.lua#L50-L56
The server should do these calculations once and then propagate them to every chip.
